backend/manager/modules/bll/src/test/java/org/ovirt/engine/core/bll/scheduling/utils/VdsCpuUnitPinningHelperTest.java

```diff
@@ -133,25 +133,37 @@ public void shouldSucceedToAllocateTwoSocketOneCoreTwoCpu() {
         vm.setThreadsPerCpu(2);
         vm.setCpuPinningPolicy(CpuPinningPolicy.DEDICATED);
         boolean result =
-                vdsCpuUnitPinningHelper.isDedicatedCpuPinningPossibleAtHost(new HashMap<>(), vm, host);
+                vdsCpuUnitPinningHelper.isDedicatedCpuPinningPossibleAtHost(new HashMap<>(), vm, host.getId());
         assertTrue(result);
     }
 
     @Test
-    public void shouldSucceedToAllocateTwoSocketThreeCoreTwoCpu() {
+    public void shouldFailToAllocateTwoSocketThreeCoreTwoCpu() {
         VM vm = new VM();
         vm.setId(Guid.newGuid());
         vm.setNumOfSockets(2);
         vm.setCpuPerSocket(3);
         vm.setThreadsPerCpu(2);
         vm.setCpuPinningPolicy(CpuPinningPolicy.DEDICATED);
         boolean result =
-                vdsCpuUnitPinningHelper.isDedicatedCpuPinningPossibleAtHost(new HashMap<>(), vm, host);
+                vdsCpuUnitPinningHelper.isDedicatedCpuPinningPossibleAtHost(new HashMap<>(), vm, host.getId());
+        assertFalse(result);
```

Comment:
great, so we also fixed a bug with this!

Reply:
indeed :)